### PR TITLE
Added Apache cache to INSTALL.md

### PR DESCRIPTION
**Pull request by ghost**
Added a template for getting a cache going with Apache2

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -175,7 +175,31 @@ It is however **strongly recommended** to also setup a server side cache to prev
...
+
+Then add this section to your Movim VirtualHost
+
+```
+CacheQuickHandler on
+CacheLock on
+CacheLockPath /tmp/mod_cache-lock
+CacheLockMaxAge 5
+CacheIgnoreHeaders Set-Cookie
+<Location />
```
Comment (edhelas) [inline INSTALL.md]: It should only be for /pictures no ?

Comment (ghost) [reply INSTALL.md]: Yep, you're correct

**Comment by edhelas** [comment]
Can you please merge the two commits and rebase the branch :) ?

**Comment by ghost** [comment]
> Can you please merge the two commits and rebase the branch :) ?

Hmm, I'm not sure how to do that through GitHub's interface and I don't have git setup on my system for this account currently

**Pull request merged**